src/test/java/com/snack/news/controller/TopicControllerTest.java

```diff
@@ -29,6 +29,9 @@
 	@InjectMocks
 	private TopicController topicController;
 
+	@Mock
+	private TopicService topicService;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused private fields such as 'topicService'.](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5523973)